src/azure-cli/azure/cli/command_modules/batch/_params.py

```diff
@@ -7,7 +7,9 @@
 
 from knack.arguments import CLIArgumentType
 
-from azure.mgmt.batch.models import AccountKeyType
+from azure.mgmt.batch.models import \
```

Comment:
Please use `()` instead of `\`. `from azure.mgmt.batch.models import (AccountKeyType, KeySource)`